gateway-js/src/executeQueryPlan.ts

```diff
@@ -322,6 +323,7 @@ async function executeFetch<TContext>(
           variables,
           error.extensions,
           error.path,
+          error,
```

Comment:
I'm aware this PR is from the previous repo and a different author, but I have just one nit. Since we're now passing down the whole error object it seems unnecessary to also pass specific properties from the same error object. We can simplify this function's signature a bit.

Reply:
Good call and not a problem! https://github.com/apollographql/federation/pull/315

Reply:
Perfect, thanks!